make/RunTests.gmk

```diff
@@ -837,6 +835,24 @@ define SetupRunJtregTestBody
     $1_JTREG_BASIC_OPTIONS += $$(addprefix $$(JTREG_PROBLEM_LIST_PREFIX), $$($1_JTREG_PROBLEM_LIST))
   endif
 
+  JTREG_ALL_OPTIONS := $$(JTREG_JAVA_OPTIONS) $$(JTREG_VM_OPTIONS)
+
+  JTREG_AUTO_PROBLEM_LISTS :=
+  JTREG_AUTO_TIMEOUT_FACTOR := 4
+
+  ifneq ($$(findstring -Xcomp, $$(JTREG_ALL_OPTIONS)), )
+    JTREG_AUTO_PROBLEM_LISTS := $$(JTREG_AUTO_PROBLEM_LISTS) ProblemList-Xcomp.txt
+    JTREG_AUTO_TIMEOUT_FACTOR ?= 10
```

Comment:
Since you assign this to be 4 a couple of lines up, this code will never do anything.

Reply:
Did you test that this patch works as intended?

Reply:
Thanks for catching.
Yes, the problem is that our system always set timeout factor. So now I also run manual testing using make to verify that correct factor is used.